docs/source/UserGuide/VUDRM-Admin.md

```diff
@@ -0,0 +1,138 @@
+
+<img src="../_static/user-guide-images/5edittokenpolicies.png" class="image" width="750" alt="VUALTO Admin - Edit Token Policies"/>
+
+Clicking `Edit Token Policy` will load the JSON editor. 
+
+<img src="../_static/user-guide-images/5.1jsoneditor.png" class="image" width="750" alt="VUALTO Admin - Edit Token Policies - JSON Editor"/>
+
+Please ensure that the JSON has been entered correctly, otherwise you will be informed of an error and will not be able to save the amended policy.
+
+When the amends are complete, click `Close` and then `Save`. 
+
+Each DRM service has limitations to how their policy can be altered. Please read the links below for more details:
+
+* [DRM Policy](../DeveloperDocumentation/VUDRM-token.html#drm-policy)
+* [PlayReady DRM Policy](../DeveloperDocumentation/VUDRM-token.html#playready-drm-policy)
+* [Fairplay DRM Policy](../DeveloperDocumentation/VUDRM-token.html#fairplay-drm-policy)
+* [Widevine DRM Policy](../DeveloperDocumentation/VUDRM-token.html#widevine-drm-policy)
+
+### VUDRM Token
+
+In this section, you can create, validate, decrypt, encode, and decode VUDRM Tokens.
+
+<img src="../_static/user-guide-images/6vudrmtoken.png" class="image" width="1000" alt="VUALTO Admin - VUDRM Token Page"/>
+
+There are a variety of templates to choose from which will automatically populate the `Policy` text box, please ensure that you replace all default values (such as `{"polend":"DD-MM-YYYY HH:MM:SS"}` to `{"polend":"09-05-2021 12:30:00"}`). When the default values have been changed, you can then generate a token with those parameters. 
+
+The buttons on the page do the following:
+
+* `Generate`        - This will generate a token to include the policy you have entered
+* `Validate Policy` - This will ensure the policy is valid
+* `Validate Token`  - Validate the token created
+* `Decrypt Token`   - Decrypts the token
+* `Encode Token`    - URL Encodes the token
+* `Decode Token`    - URL Decodes the token
+
+Once a token has been generated, a new button will appear - `Use Token in Demo`. 
+
+<img src="../_static/user-guide-images/6-1vudrmtoken.png" class="image" width="1000" alt="VUALTO Admin - VUDRM Token Page - Use Token in Demo"/>
+
+Clicking `Use Token in Demo` will load the `VUPLAY Demo` player with the generated token in the `Custom VUDRM Token` text box.
+
+For an in depth guide of how our tokens work - please refer to the [VUDRM Token documentation](../DeveloperDocumentation/VUDRM-token.html#). 
+
+### VUDRM Encryption Keys
+
+Within this section, you can call the `CPIX Key Provider` to fetch VUDRM Encryption Keys in CPIX XML document format as well as the CPIX keys as JSON which is generated at the same time, both of which are available to copy. 
+
+We also have our `Legacy JSON Key provider` (available on the `Key Provider API` dropdown) to use if your product requires it.
```

Comment:
Does this line need to be here if its also line 95?

Reply:
Nope. Good spot!